com.ibm.streamsx.topology/opt/python/include/splpy_funcop.h

```diff
@@ -38,10 +38,20 @@ class SplpyFuncOp : public SplpyOp {
          addAppPythonPackages();
          loadAndWrapCallable(wrapfn);
       }
+
+    SplpyFuncOp(SPL::Operator * op, const std::string & wrapfn,
+		const std::string & module, const std::string & function) :
+         SplpyOp(op, "/opt/python/packages/streamsx/topology")
+      {
+         addAppPythonPackages();
```

Comment:
Isn't this duplicate code, e.g. the `addAppPythonPackages` and `loadAndWrapCallable(wrapfn)` will be called by the constructor for `SplpyOp` ?